### PR TITLE
#327 Added pass phrase hint

### LINKED ISSUE #327: add a modal with hint about how to choose pass phrase
There should be a little question mark icon at the end of pass phrase creation prompt texts like `Choose your pass phrase (?)`. When clicked, this shows:

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR added pass phrase hint modal

close #327 // if this PR closes an issue

image

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by ioanmo226** [comment]
@sosnovsky I used the simplest solution here. Just used html(copied from Android source code) and used webview.
Let me know if you want more advanced solution using native solution or something like that.

**Comment by tomholub** [comment]
I'd go with simple, but can the text be like twice the size? It looks puny

**Comment by tomholub** [comment]
Since that may make the pass phrases stretch over one line, let's also add another blank line in between the pass phrases

**Comment by sosnovsky** [comment]
Can we convert html to `NSAttributedString` and show it in `UILabel`? Probably should be even simpler and it's a native way

**Comment by ioanmo226** [comment]
Ok, got it
I used webview because I even didn't have to create new view controller :-)

**Comment by ioanmo226** [comment]
@sosnovsky Please check
By the way, has someone touched semaphoreci env?
image

**Comment by sosnovsky** [comment]
> @sosnovsky Please check
> 
> By the way, has someone touched semaphoreci env?
> 
> image
> 
> 

The same on the flowcrypt-browser repo - https://github.com/FlowCrypt/flowcrypt-browser/issues/5343

File: FlowCrypt/Resources/pass_phrase_hint.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<!--
+  ~ © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+  ~ Contributors: DenBond7
+  -->
+<style>
+    body {
+        font-size: 60px;
```
Comment (sosnovsky) [inline FlowCrypt/Resources/pass_phrase_hint.html:7]: let's use `font-size: 48px`, as 60px looks too big in comparison with our common UI fonts

File: FlowCrypt/Resources/pass_phrase_hint.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <span class="good_passphrase">semi automated shiseido mask</span>
+        <span class="good_passphrase">icelanders prefer real sauna</span>
+        <p>Combinations like that are memorable, easy to type and the most secure.</p>
+        <p>In contrast, pass phrases like <code class="bad">@ut0m4t3d</code>, <code class="bad">Honey167!</code> or
+            <code class="bad">lA45pf8$d</code> are a poor choice when it comes to security. Because they are so short,
+            on a fleet of 10,000 ordinary computers, these <b>can be guessed in 7 seconds</b>.
+        </p>
+        <p>To read more, see <a
+                href="https://blogs.dropbox.com/tech/2012/04/zxcvbn-realistic-password-strength-estimation/"
+                target="_blank">zxcvbn article</a> by Dropbox.</p>
```
Comment (sosnovsky) [inline FlowCrypt/Resources/pass_phrase_hint.html:48]: tapping on link do nothing, probably we can just remove it for simplifying implementation

File: FlowCrypt/Controllers/Setup/SetupViewDecorator.swift [hunk 3]
```diff
@@ -80,7 +80,7 @@ struct SetupViewDecorator {
         case .common:
             subtitle = "setup_description".localized
         case .choosingPassPhrase:
-            subtitle = "" // "create_pass_phrase_description".localized https://github.com/FlowCrypt/flowcrypt-ios/issues/497
+            subtitle = "create_pass_phrase_description".localized
```
Comment (sosnovsky) [inline FlowCrypt/Controllers/Setup/SetupViewDecorator.swift:83]: `See choosing secure pass phrase ?` looks strange, at first I thought that it's incorrect localisation string, and it's not clear that you can tap on it.

Let's change text to `Tap to see tips for choosing secure pass phrase`

File: FlowCrypt/Resources/pass_phrase_hint.html (hunk 1 above)
Comment (ioanmo226) [reply FlowCrypt/Resources/pass_phrase_hint.html:7]: ok

**Review by sosnovsky — APPROVED** [review]
👍

**Pull request merged**